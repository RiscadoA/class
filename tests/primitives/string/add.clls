proc main() {
    println("Hello" + ", " + "World!");
    print("A" + "B" + "C" + "\n");
    println("A new line");
    println("A boolean: " + true);
    println("An integer: " + 42);
    println(false + " is false");
    println(42 + " is the answer");
    ()
};;
