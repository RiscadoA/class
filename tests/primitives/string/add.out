Hello, World!
ABC
A new line
A boolean: true
An integer: 42
false is false
42 is the answer
